### PR TITLE
[HTP-65] Quantcast: New Adapter

### PR DESCRIPTION
**Pull request by amir-sabbaghi**
## Type of Change
- [x] New adapter
- [ ] Feature Update
- [ ] Bug Fix
- [ ] Code Refactor
- [ ] Other

## Description of Change
Initial version of Quantcast adapter.

### CODE REVIEW

File: quantcast/quantcast-htb-system-tests.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    };
+}
+
+function getCallbackType() {
+    return 'NONE';
+}
+
+function getArchitecture() {
+    return 'MRA';
+}
```
Comment (ShreeniwasIyer) [inline quantcast/quantcast-htb-system-tests.js]: Can't we use Partner.CallbackTypes.NONE and Partner.Architectures.MRA here?

Comment (amir-sabbaghi) [reply quantcast/quantcast-htb-system-tests.js]: No. We can't do that. `Partner.CallbackTypes.NONE` and `Partner.Architectures.MRA` are numbers whereas these two functions must return string.

**Comment by ix-certification** [comment]
Please fix the following items:
1. Put HTP ticket number in the pull request title.
2. Remove `EventsService` variable that is not in use now in this line. https://github.com/amir-sabbaghi/ht-wrapper-adapters/blob/qc/quantcast/quantcast-htb.js#L18
3. Fix all of the lint errors. Refer to this page for instructions on how to run lint. https://kb.indexexchange.com/Adapters/ESLint.htm

**Comment by ix-certification** [comment]
Please add all of the metascript (comment blocks that start with `//?`) back as to how they are used in the template. For example, in the template, we have metascript around `parcel.price`. Please add them back.  

```
//? if (FEATURES.RETURN_PRICE) {
	parcel.price = Number(__baseClass._bidTransformers.price.apply(bid.cpm));
//? }
```

**Comment by amir-sabbaghi** [comment]
I didn't see them being anywhere. So I removed them.
Anyway it's fixed now.

**Pull request merged**